ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -59,9 +59,34 @@ define([
             propertyJSON.help =  'The distance (number of words) within which search terms must be found in order to match';
             delete propertyJSON.enum;
         }
+
+        // if we don't set this the property model will transform the value as if it's a date, clobbering the duration format
+        if (comparator === 'RELATIVE') {
+            propertyJSON.transformValue = false;
+        }
+
         return propertyJSON;
     }
 
+    // Relative time view is needed for the relative date, multivalue for all else
+    const determineView = (comparator) => {
+        let necessaryView;
+        switch(comparator) {
```

Comment:
✏️ seems overkill to have a `switch` here since it's only one predicate (plus an inverse) that's tested.

Reply:
Yeah, I'm expecting this to have more in the future.  I can make it an if statement or ternary for now if people would prefer.